.github/workflows/tpu-tests.yml

```
@@ -1,3 +1,5 @@
+name: PL tests on TPU
```

Maybe, something like this?
```suggestion
name: Test PyTorch on TPU
```
to be consistent with other workflow names:
```console
$ grep -rnI "^name: " .github/workflows|grep test
.github/workflows/ci-pytorch-test-full.yml:1:name: Test PyTorch full
.github/workflows/ci-lite-test-full.yml:1:name: Test Lite full
.github/workflows/ci-app-cloud-e2e-test.yml:1:name: cloud-e2e-testing
.github/workflows/ci-pytorch-test-slow.yml:1:name: Test PyTorch slow
.github/workflows/ci-app-tests.yml:1:name: Test App
```